allen-cell-animated/colorizer-data
# refactor: Python packaging

Closes #11 !

- Refactors `data_writer_utils`, which has been split into a `writer` and `utils` module within the new `colorizer_data` package.
- Updates documentation with instructions on installing and updating the project.

Testing:
1. Create a clean conda environment.
2. Run `pip install git+https://github.com/allen-cell-animated/colorizer-data.git@refactor/python-package`.
3. Run `python` to open the interactive environment.
4. Paste in the following lines. It should run without errors.
```
from colorizer_data.writer import (
    ColorizerDatasetWriter,
)
from colorizer_data.utils import generate_frame_paths

writer = ColorizerDatasetWriter(".", "", 1)
print(writer)

print(generate_frame_paths(34))
```
